packages/flask/src/UICarouselView/types.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Type of UICarouselContainer children
+ */
+export type CarouselChildren = React.ReactElement<UICarouselViewPageProps>[] | React.ReactElement<UICarouselViewPageProps>
+| React.ReactElement<UICarouselViewPageProps>[];
+/**
+ * Container properties.
+ */
+export type UICarouselViewContainerProps = {
+   /** callback that is called when the page changes */
+   onPageIndexChange?: (newPageIndex: number) => void;
+   /** determines whether pages should move when clicked */
+   isPageMovesOnPress?: boolean
```

Comment:
This is a bit cleaner in English
```suggestion
   shouldPageMoveOnPress?: boolean
```